Cars/Cars/Controllers/CarsController.cs

```diff
@@ -25,10 +25,40 @@ public IActionResult List()
         {
             ViewBag.Title = "Auto Veo";
             CarsListViewModel Obj = new CarsListViewModel();
-            Obj.GetAllCars = _allCars.Cars;
+            Obj.GetAllCars = _Car.Cars;
             Obj.CurrentCategory = "Autos";
-                   
+
             return View(Obj);
         }
+
+        public ViewResult GermanCars()
```

Comment:
all these three methods should be replaced with **public ViewResult Cars(forType: SomeType)** or similar